src/storm/modelchecker/abstraction/AbstractAbstractionRefinementModelChecker.cpp

```diff
@@ -89,7 +89,8 @@ std::unique_ptr<CheckResult> AbstractAbstractionRefinementModelChecker<ModelType
 
 template<typename ModelType>
 std::unique_ptr<CheckResult> AbstractAbstractionRefinementModelChecker<ModelType>::computeReachabilityRewards(
-    Environment const& env, storm::logic::RewardMeasureType rewardMeasureType, CheckTask<storm::logic::EventuallyFormula, ValueType> const& checkTask) {
+    Environment const& env, [[maybe_unused]] storm::logic::RewardMeasureType rewardMeasureType,
```

Comment:
The rewardMeasureType should not be unused, someone should look at this and fix it